app/frontend/wallet/shelley/shelley-trezor-crypto-provider.ts

```diff
@@ -373,10 +376,10 @@ const ShelleyTrezorCryptoProvider = async ({
         return {
           catalystRegistrationParameters: {
             votingPublicKey: txAuxiliaryData.votingPubKey,
-            stakingPath: txAuxiliaryData.rewardDestinationAddress.stakingPath,
+            stakingPath: txAuxiliaryData.rewardDestinationAddress.stakingPath || '',
```

Comment:
rather than fallback to empty string which if triggered would break the code unpredictably, please put here a runtime assertion